src/Analysis/Codelyzer.Analysis.Model/ConstructorDeclaration.cs

```diff
@@ -0,0 +1,23 @@
+﻿using System.Collections.Generic;
+using Newtonsoft.Json;
+ 
+namespace Codelyzer.Analysis.Model
+{
+    public class ConstructorDeclaration : UstNode
```

Comment:
Can we use this class to extend MethodDeclaration - in addition, the method will have two more variables.

Reply:
I have extended the MethodDeclaration class with ConstructorDeclaration, but could you elaborate on the two method variables that are required?